src/pymovements/utils/archives.py

```diff
@@ -48,21 +48,21 @@ def extract_archive(
     ----------
     source_path : Path
         Path to the file to be extracted.
-    destination_path : Path, optional
+    destination_path : Path | None
         Path to the directory the file will be extracted to. If omitted, the directory of the file
         is used.
     recursive : bool
         Recursively extract archives which are included in extracted archive.
     remove_finished : bool
         If ``True``, remove the file after the extraction.
-    verbose:
+    verbose : int
         Verbosity levels: (1) Print messages for extracting each dataset resource without printing
         messages for recursive archives. (2) Print additional messages for each recursive archive
         extract.
 
     Returns
     -------
-    Path :
+    Path : Path
```

Comment:
is there a reason for naming this return value Path?

is pydoclint complaining about the line otherwise?

maybe without the `:`?